### PR TITLE
improve documentation about document keys

### PR DESCRIPTION
**Pull request by jsteemann**
Following a suggestion of @joerg84

### CODE REVIEW

File: 3.10/data-modeling-operational-factors.md [hunk 1]
```diff
@@ -274,6 +274,30 @@ Consider some ways to minimize the required amount of storage space:
...
+- Explicitly set the `_key` field to a custom unique value.
+  This enables you to store information in the `_key` attribute instead of another
+  attribute inside the document. The `_key` value is always indexed, so it is
+  preferrable to use it rather than to store the document ids in another attribute
+  and creating an extra index on it. 
+- Try to use short values for the `_key` attribute. 
+  The `_key` values will be used whenever a document is looked up by its primary
+  key, and shorter key values can improve lookup performance and reduce disk usage. 
+  As the `_key` values are also used a foreign keys in the `_from` and `_to` attributes 
+  of edges, the key length also matters all for graph operations. Again, shorter keys
```
Comment (neunhoef) [inline 3.10/data-modeling-operational-factors.md:289]:
```suggestion
  of edges, the key length also matters for all graph operations. Again, shorter keys 
```

File: 3.11/data-modeling-operational-factors.md [hunk 2]
```diff
@@ -274,6 +274,30 @@ Consider some ways to minimize the required amount of storage space:
...
+- Explicitly set the `_key` field to a custom unique value.
+  This enables you to store information in the `_key` attribute instead of another
+  attribute inside the document. The `_key` value is always indexed, so it is
+  preferrable to use it rather than to store the document ids in another attribute
+  and creating an extra index on it. 
+- Try to use short values for the `_key` attribute. 
+  The `_key` values will be used whenever a document is looked up by its primary
+  key, and shorter key values can improve lookup performance and reduce disk usage. 
+  As the `_key` values are also used a foreign keys in the `_from` and `_to` attributes 
+  of edges, the key length also matters all for graph operations. Again, shorter keys
```
Comment (neunhoef) [inline 3.11/data-modeling-operational-factors.md:289]:
```suggestion
  of edges, the key length also matters for all graph operations. Again, shorter keys 
```

File: 3.9/data-modeling-operational-factors.md [hunk 3]
```diff
@@ -274,6 +274,30 @@ Consider some ways to minimize the required amount of storage space:
...
+- Explicitly set the `_key` field to a custom unique value.
+  This enables you to store information in the `_key` attribute instead of another
+  attribute inside the document. The `_key` value is always indexed, so it is
+  preferrable to use it rather than to store the document ids in another attribute
+  and creating an extra index on it. 
+- Try to use short values for the `_key` attribute. 
+  The `_key` values will be used whenever a document is looked up by its primary
+  key, and shorter key values can improve lookup performance and reduce disk usage. 
+  As the `_key` values are also used a foreign keys in the `_from` and `_to` attributes 
+  of edges, the key length also matters all for graph operations. Again, shorter keys
```
Comment (neunhoef) [inline 3.9/data-modeling-operational-factors.md:289]:
```suggestion
  of edges, the key length also matters for all graph operations. Again, shorter keys 
```

**Review by neunhoef — APPROVED** [review]
LGTM. Just one typo in three places. See suggestion.

**Review by Simran-B — APPROVED** [review]

**Pull request merged**